nltk/corpus/reader/extended_omw.py

```diff
@@ -0,0 +1,137 @@
+# Natural Language Toolkit: Exteneded Open Multilingual WordNet Reader
```

Comment:
```suggestion
# Natural Language Toolkit: Extended Open Multilingual WordNet Reader
```
I haven't had the time to take a full look at this PR yet, but I noticed this small typo.